### ISSUE TITLE
getAllChromosomesBed fails with KeyError: 'HLA-A*01'

### ISSUE THREAD

**Issue by stfacc** [epi2me-labs/wf-human-variation#138]
### Operating System

Ubuntu 22.04

### Workflow Version

v1.10.1-g79ddc44

### Workflow Execution

Command line

### CLI command run

nextflow run epi2me-labs/wf-human-variation --bam /home/sfacchini/mnt/rd013s/230813_Andrea_WGS_analysis/bam/61552/61552.bam --ref /data/array/sfacchini/wf-human-variation_test/Homo_sapiens_assembly38.fasta --sample_name sample_61552 --snp --sv --cnv --str --phased --sv_benchmark --threads 64

### Workflow Execution - CLI Execution Profile

standard (default)

### What happened?

Workflow does not complete.

### Relevant log output

```shell
Workflow execution completed unsuccessfully!

The exit status of the task that caused the workflow execution to fail was: 1.

The full error message was:

Error executing process > 'getAllChromosomesBed (1)'

Caused by:
  Process `getAllChromosomesBed (1)` terminated with an error exit status (1)

Command executed:

  faidx --transform bed Homo_sapiens_assembly38.fasta > allChromosomes.bed

Command exit status:
  1

Command output:
  (empty)

Command error:
  Traceback (most recent call last):
    File "/home/epi2melabs/conda/bin/faidx", line 10, in 
      sys.exit(main())
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pyfaidx/cli.py", line 202, in main
      write_sequence(args)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pyfaidx/cli.py", line 51, in write_sequence
      outfile.write(transform_sequence(args, fasta, name, start, end))
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/pyfaidx/cli.py", line 121, in transform_sequence
      line_len = fasta.faidx.index[name].lenc
  KeyError: 'HLA-A*01'

Work dir:
  /data/array/sfacchini/wf-human-variation_test/work/ef/666925c42764353af4683d525c4f8c

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh
```

### Were you able to successfully run the latest version of the workflow with the demo data?

yes

**Comment by vlshesketh** [comment]
Hi @stfacc thank you for reporting this! We will investigate a fix and update when it's released.

**Comment by vlshesketh** [comment]
Hi @stfacc, this has been fixed in [v1.11.0](https://github.com/epi2me-labs/wf-human-variation/releases/tag/v1.11.0) which we have just released. Thank you for using wf-human-variation, and please let us know if you have any further issues.
